src/schemaHelper.js

```diff
@@ -23,8 +23,8 @@ export const formSchemaMixin = {
     },
     methods: {
         onChange(value) {
-            this.inputValue = value;
-            this.$emit('onChange', this.name, value);
+            this.inputValue = value;                
+            this.$emit('onChange', this.isMulti ? this.identifier: this.name, value);
```

Comment:
Why not just pass a `name` prop to the component from the multi input? now you need 2 extra props, just for this.